test/integration/fixtures/fixtures.go

```diff
@@ -480,7 +480,13 @@ func getExpectedAuthorizationPolicy(serviceName string, modifications []func(*se
 						Source: &securityV1beta1.Source{
 							Principals: []string{
 								"user/sa/foo",
+								"athenz.cloud/ns/user/sa/user.foo",
+								"athenz.cloud/ns/user/sa/user.foo",
```

Comment:
For users, we would only need to add `"user/sa/foo"` and `"athenz.cloud/ns/default/sa/user.foo"` values.

Reply:
In this scenario, that is not duplication, user is considered as Domain, so we have two SPIFEE ID's, one referring user to domain and the other referring user to namespace